src/main/java/edu/ucsb/mapache/controllers/SearchHistoryController.java

```diff
@@ -63,20 +63,28 @@ public DecodedJWT getJWT(String authorization) {
         if (!authControllerAdvice.getIsMember(authorization))
             return getUnauthorizedResponse("member");
         
-       if (!authControllerAdvice.getIsAdmin(authorization)) {
+      
```

Comment:
(This is a comment for lines 61-64, not line 66)

The method to get all user searches should be restricted to only admins. Members should not have access to this endpoint.

Additionally, now that there are separate endpoints for getting a specific user's searches versus getting the searches of all members, it would be nice if we could change the name of this method to be more descriptive. For example, `getAllUserSearches` might be a better name.

Reply:
Add new condition statement to check if the user is admin and change the function name.

Reply:
Can you change the method name to follow camelCase naming conventions? Like `getAllUserSearches`